capstone-frontend/src/LeftSideBar.test.js

```diff
@@ -13,8 +13,14 @@ import {
 const container = document.createElement("div");
 document.body.appendChild(container);
 
+const getUserBookLists = () => {
+  return [{ id: "1", name: "Best Books" },
+  { id: "2", name: "Fantasy" },
+  { id: "3", name: "Sci-fi" }];
```

Comment:
nit: Use single quotes for string literals (https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes).

This applies everywhere but I'm just going to make the one comment here.

Reply:
Ok